apps/docs/content/docs/api-references/cli-api.mdx

```diff
@@ -171,81 +182,92 @@ Tailwind CSS settings have been updated in: /project-path/tailwind.config.js
 ✅ Components added successfully
 ```
 
-## upgrade
+### Upgrade
 
 Upgrade the NextUI components to the latest version.
 
 ```codeBlock bash
 nextui upgrade [components...] [options]
 ```
 
-### Options
+#### Upgrade Options
+
+- `-p --packagePath` [string] The path to the package.json file
+- `-a --all` [boolean] Upgrade all the NextUI components (default: `false`)
+- `-w --write` [boolean] Write the upgrade commands to a file instead of executing them (default: `false`)
+- `-h, --help` Display help for command
```

Comment:
there is a comma after `-h` while other flags don't have. Better keep it consistent.

Reply:
Got